obs2ioda-v2/src/cxx/netcdf_utils.cc

```
@@ -0,0 +1,34 @@
+    std::shared_ptr<netCDF::NcGroup> getRootGroup(
+            int netcdfID,
+            const char *groupName
+    ) {
+        auto file = FileMap::getInstance().getFile(netcdfID);
+        if (groupName != nullptr) {
+            return std::make_shared<netCDF::NcGroup>(file->getGroup(groupName));
+        }
+        return file;
+    }
+
+    std::string removeWhiteSpace(
```

cppcheck (2.10-2)

[style] The function 'removeWhiteSpace' is never used.

(unusedFunction)